src/components/viewport/TransformToolbar.js

```diff
@@ -8,15 +10,318 @@ var TransformButtons = [
   { value: 'rotate', icon: 'fa-repeat' }
 ];
 
+function getCookie(name) {
```

Comment:
As mentioned off line, if this is copied logic from elsewhere, it should be removed from that location. If we can export from other file and import here, that would be more ideal so we don't need to review every line here again